### ISSUE TITLE
Ctrl-Enter does not close tray (Monaco)

### ISSUE THREAD

**Issue by hazymat** [node-red/node-red#4377]
Version 3.1.0 - **"Pressing CTRL-Enter when monaco editor has focus, does not operate the Done button (accept changes and close the tray)"**

This issue was previously resolved (refer to [Issue 3093](https://github.com/node-red/node-red/issues/3093)) but seems to have re-appeared in 3.1.0.

Reported recently [here](https://discourse.nodered.org/t/node-red-3-1-0-released/81078/13?u=hazymat) (forum thread for 3.1.0)

**Comment by Steve-Mcl** [comment]
@hazymat 


Since the previous fix, it seems Monaco have changed the internal API (thats how it goes) but replacing the code

```js
        //Unbind ctrl-Enter (default action is to insert a newline in editor) This permits the shortcut to close the tray.
        try {
            ed._standaloneKeybindingService.addDynamicKeybinding(
                '-editor.action.insertLineAfter', // command ID prefixed by '-'
                null, // keybinding
                () => {} // need to pass an empty handler
            );
        } catch (error) { }
```

in file `packages/node_modules/@node-red/editor-client/src/js/ui/editors/code-editors/monaco.js`

with this code:

```js
        //Unbind ctrl-Enter (default action is to insert a newline in editor) This permits the shortcut to close the tray.
        try {
            monaco.editor.addKeybindingRule({keybinding: 0, command: "-editor.action.insertLineAfter"});
        } catch (error) {
            console.warn(error)
        }
```

does prevent monaco stealing the `ctrl+enter` shortcut.

Feel free to raise a PR (against master branch)

If not, i will do so when I get 10 mins free.

**Comment by Steve-Mcl** [comment]
NOTE: Ideally, Unbinding `ctrl-Enter` would only be called when `ctrl+enter` at "editor scope" is not bound to anything in node-red shortcuts.

**Comment by hazymat** [comment]
I have forked node-red and created a PR, although I do get a message on that PR from [linux-foundation-easycla](https://github.com/apps/linux-foundation-easycla) that "The commit (https://github.com/node-red/node-red/commit/44c0bbc61e052f20077aa0755114d86b4efacc7f) is not authorized under a signed CLA"

I'm not sure what this means, maybe someone might take a look?

**Comment by hazymat** [comment]
Have raised a PR

**Comment by knolleary** [comment]
Fixed by #4377
